backends-clickhouse/src/main/scala/io/glutenproject/backendsapi/clickhouse/CHSparkPlanExecApi.scala

```diff
@@ -189,7 +189,6 @@ class CHSparkPlanExecApi extends ISparkPlanExecApi {
       splitTime: SQLMetric,
       spillTime: SQLMetric,
       compressTime: SQLMetric,
-      prepareTime: SQLMetric,
```

Comment:
Do we need to count it? what it's counted for?

Reply:
No, it is counting the time shuffle write input batch(ArrowBatch) to ArrowArray and ArrowSchema